docs/self-hosted/config-options.md

```diff
@@ -339,6 +339,17 @@ WebSockets work reliably across multiple containers of Directus.
 
 Redis is required when you run Directus load balanced across multiple containers/processes.
 
+### Data Retention
+
+| Variable              | Description                                                                                                      | Default Value |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------- | ------------- |
+| `RETENTION_ENABLED`   | Whether or not to enable custom data retention settings. `false` will not delete data.       |
```

Comment:
the formatting is a bit off here, and one `|` seems to be missing  🔎 
since the default seems to be in fact `false`, we should wrote it in the docs, shoudnt't we? 🤔